src/User.php

```diff
@@ -8,11 +8,7 @@
  */
 namespace Corcel;
 
-use Illuminate\Database\Eloquent\Model as Eloquent;
-use Illuminate\Database\Eloquent\Relations\HasMany;
-use Illuminate\Database\Eloquent\Relations\BelongsToMany;
-
-class User extends Eloquent
+class User extends Model
```

Comment:
Why did you remove `implements Authenticatable, CanResetPassword` in `User` declaration? The users' test are not passing :-(